docs/userguide/objective.md

```diff
@@ -1,67 +1,92 @@
 # Objective
 
-The [`Objective`](baybe.objective.Objective) instructs BayBE how to deal with multiple
-targets (if applicable).
-
-To create an objective, it is necessary to provide the following:
-* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
-  denoting the optimization of a single target function or a combination of
-  different target functions respectively.
-* A list of `targets`: The list of targets that are optimized (see user guide for
-  [`Target`](../../userguide/targets)). Note that the `SINGLE` mode also requires a
-  list containing the single target.
+Optimization problems involve either a single target quantity of interest or 
+several (potentially conflicting) targets that need to be considered simultaneously.  
+BayBE uses the concept of an [`Objective`](baybe.objective.Objective) to allow the user
+to control how these different types of scenarios are handled.
 
 ```{note}
-We are actively working on adding more objective modes for multiple targets.
+We are actively working on adding more objective types for multiple targets.
```

Comment:
And in fact, we are in a sense already working on it because I am currently refactoring stuff for the scaling, which is directly connected ;) so I'd say let's keep it 🙃